core/main/src/firebolt/handlers/audio_description_rpc.rs

```diff
@@ -0,0 +1,70 @@
+// Copyright 2023 Comcast Cable Communications Management, LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+use jsonrpsee::{
+    core::{async_trait, RpcResult},
+    proc_macros::rpc,
+};
+use ripple_sdk::api::{
+    device::device_accessibility_data::AudioDescriptionSettings,
+    gateway::rpc_gateway_api::CallContext, storage_property::StorageProperty,
+};
+
+use crate::{
+    processor::storage::storage_manager::StorageManager, state::platform_state::PlatformState,
+};
+
+#[rpc(server)]
+pub trait AudioDescription {
+    #[method(name = "AudioDescriptions.enabled", aliases=["accessibility.audioDescriptionSettings"])]
```

Comment:
I might be wrong but I thought `accessibility.audioDescriptionSettings` returned more than just whether or not AD is enabled?

Reply:
```
"AudioDescriptionSettings": {
						"title": "AudioDescriptionSettings",
						"type": "object",
						"required": [
							"enabled"
						],
						"properties": {
							"enabled": {
								"type": "boolean",
								"description": "Whether or not audio descriptions should be enabled by default"
							}
						}
					}
```

Reply:
but that returns an object with a boolean in it, instead of just a boolean.
Think they need to be separate methods